requirements.txt

```diff
@@ -1,5 +1,5 @@
 PyQt5==5.12.3
```

Comment:
PyQt5==5.15.9 in https://github.com/AllYarnsAreBeautiful/ayab-desktop/pull/572 ?